packages/replayio/src/utils/getSystemOpenCommand.ts

```diff
@@ -0,0 +1,10 @@
+export function getSystemOpenCommand() {
+  switch (process.platform) {
+    case "darwin":
+      return "open";
+    case "linux":
+      return "xdg-open";
+    default:
+      throw new Error("Unsupported platform");
```

Comment:
brb, I'm calling Mark ☎️ 😉  ChatGPT suggests that `start` could work in case of `win32` but I guess that maybe we don't support windows chromium atm?

Reply:
Yeaaah TBH I don't know. The old CLI threw in this case, so I carried the behavior over. I guess I could toss `start` in here though and at least we'd be one step closer.